### PR TITLE
A welcome page

### PR DESCRIPTION
**Pull request by mistercrunch**
It should grow into having more controls, but this looks pretty good already.
screen shot 2016-03-25 at 6 09 52 pm

### CODE REVIEW

File: dashed/assets/javascripts/welcome.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+var $ = window.$ = require('jquery');
+var jQuery = window.jQuery = $;
+var px = require('./modules/dashed.js');
+
+require('../stylesheets/dashed.css');
+require('../stylesheets/welcome.css');
+
+require('bootstrap');
+require('datatables');
+require('d3');
```
Comment (williaster) [inline dashed/assets/javascripts/welcome.js]: does this export a global `d3` var for the calendar? i.e., does it not need to be `var d3 = require('d3');`?

File: dashed/assets/javascripts/welcome.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+require('../node_modules/cal-heatmap/cal-heatmap.css');
+var CalHeatMap = require('cal-heatmap');
+
```
Comment (williaster) [inline dashed/assets/javascripts/welcome.js]: multiple spaces shouldn't pass lint :P

File: dashed/assets/javascripts/welcome.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  cal.init({
+    start: new Date().setFullYear(new Date().getFullYear() - 1),
+    range: 13,
+    data: '/dashed/activity_per_day',
+    domain : "month",
+    subDomain : "day",
+    itemName: "action",
+    tooltip: true
+  });
+  modelViewTable('#dash_table', '/dashboardmodelviewasync');
```
Comment (williaster) [inline dashed/assets/javascripts/welcome.js]: just something to note: every time you give a DOM element an `id`, it creates a GLOBAL javascript variable with that id name. there are tons of instances of using `#my_id` selectors in the code base so it's probably not worth changing at this point, but going forward it is better to use `class`es than `id`s for DOM elements and selections.

File: dashed/assets/stylesheets/dashed.css [hunk 4]
```diff
@@ -68,6 +68,10 @@ form div {
 }
 .navbar-brand a {
   color: white;
+  text-decoration: none;
```
Comment (williaster) [inline dashed/assets/stylesheets/dashed.css]: thank god!

File: dashed/models.py [hunk 5]
```diff
@@ -68,6 +69,22 @@ def created_by_(self):
     def changed_by_(self):
         return '{}'.format(self.changed_by or '')
 
+    @property
+    def modified(self):
+        s = humanize.naturaltime(datetime.now() - self.changed_on)
+        return "<nobr>{}</nobr>".format(s)
```
Comment (williaster) [inline dashed/models.py]: I removed these from the dashboard page, they are non-standard DOM elements and are not on the roadmap to be supported. maybe use something else? `<span>` or something?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

**Comment by williaster** [comment]
couple nit things but overall looks good :)

File: dashed/assets/javascripts/welcome.js (hunk 1 above)
Comment (mistercrunch) [inline dashed/assets/javascripts/welcome.js]: it seems to work after removing both dash.js (which I don't need) and d3... I'll keep it off then

File: dashed/assets/javascripts/welcome.js (hunk 3 above)
Comment (mistercrunch) [inline dashed/assets/javascripts/welcome.js]: oh I didn't know that... will avoid id moving forward, a unique class works just as well

File: dashed/models.py (hunk 5 above)
Comment (mistercrunch) [inline dashed/models.py]: oh good to know. no-no-br! I fixed it with a css class :
```
.no-wrap {
    white-space: nowrap;
}
```

**Comment by mistercrunch** [comment]
Fixed everything, merging!

**Pull request merged**